src/Command/DispatchCommand.php

```diff
@@ -53,6 +52,15 @@ final class DispatchCommand extends AbstractNeedApplyCommand
      */
     private $projects;
 
+    public function __construct(GitWrapper $gitWrapper, Filesystem $fileSystem, Environment $twig)
+    {
+        parent::__construct();
+
+        $this->gitWrapper = $gitWrapper;
```

Comment:
= new GitWrapper() fix the issue locally. But I didnt find a good enough internet connection to push 😅

Reply:
Well it could be binded manually on DI or it is not on the container?

Reply:
I don't think it's on the container. I created https://github.com/sonata-project/dev-kit/pull/800 then.